docker/docker-download-xs.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -ex
+
+# Download NNDC HDF5 data
+if [[ ! -e $HOME/nndc_hdf5/cross_sections.xml ]]; then
+    wget https://anl.box.com/shared/static/a0eflty17atnpd0pp7460exagr3nuhm7.xz -O - | tar -C $HOME -xvJ
```

Comment:
I still have the same objection in that these URLs appear in two different files. Maybe we should just call this one `tools/ci/download-xs.sh` and then call it from `tools/ci/travis-before-script.sh` (as well as from the Dockerfile)